src/proposal.ts

```diff
@@ -511,25 +511,46 @@ export class Proposal implements IStateful<IProposalState> {
       const threshold = realMathToNumber(new BN(item.gpQueue.threshold))
       const stakesFor = new BN(item.stakesFor)
       const stakesAgainst = new BN(item.stakesAgainst)
+      /**
+       * for doing multiplication between floating point (threshold) and BN numbers
+       */
+      const PRECISION = Math.pow(2, 40)
 
-      // upstakeNeededToPreBoost is the amount of tokens needed to upstake to move to the preboost queue
-      // this is only non-zero for Queued proposals
-      // note that the number can be negative!
+      /**
+       * The number of up-staking tokens (usually GEN) needed to qualify a queued proposal to move into the
+       * pre-boosted queue.
+       *
+       * Only computed for queued proposals.
+       *
+       * The equation is derived from: threshold = (stakesFor + upstakeNeededToPreBoost) / stakesAgainst
+       *
+       * >=  0 : any number of up-staking tokens greater than this will qualify to move the proposal
```

Comment:
I do not really get what "this" denotes in these lines. What is `>=0` ?

I'd just remove them: I think the line above ("the number of up-stsakeing tokens needed to qualify...") explains it pretty well already.

Reply:
I clarified the comments.  I think they provide more info than is readily available by thinking about the equations.